cookbook/Berksfile

```diff
@@ -0,0 +1,9 @@
+source 'https://supermarket.getchef.com'
+
+metadata
+
+cookbook 'java'
+cookbook 'et_nginx'
+cookbook 's3_file', path: '../../../evertrue/cookbooks/s3_file'
+cookbook 'zookeeper',
+         path: '../../chef-zookeeper'
```

Comment:
Both of these `path` style references are EverTrue specific, and additionally, if these cookbooks are only used for integration, should be placed inside a `group :integration`.